WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -289,7 +299,7 @@ private ISearchItem[] CreateSearchItems()
 
 	private ISearchItem CreateSendItem()
 	{
-		return new ActionableItem("Send", "Display wallet send dialog", () => { SendCommand.ExecuteIfCan(); return Task.CompletedTask; }, "Wallet", new[] { "Wallet", "Send", "Action", }) { Icon = "wallet_action_send", IsDefault = true, Priority = 1 };
+		return new ActionableItem("Send", "Display wallet send dialog", () => { DefaultSendCommand.ExecuteIfCan(); return Task.CompletedTask; }, "Wallet", new[] { "Wallet", "Send", "Action", }) { Icon = "wallet_action_send", IsDefault = true, Priority = 1 };
```

Comment:
good. receive should also have the default command

Reply:
Nice catch, added